doc/scripts/extract_content.py

```diff
@@ -90,10 +90,21 @@ def src_deps(zephyr_base, src_file, dest):
             continue
 
         dep_rel = m.group('dep_rel')  # relative to src_dir
+        # Try a relative path first
         dep_src = path.abspath(path.join(src_dir, dep_rel))
         if not path.isfile(dep_src):
+            # Not a relative path, check if it's absolute if we have been
+            # provided with a sphinx source directory root
+            if path.isabs(dep_rel) and src_root:
```

Comment:
Why not do this check first instead? If it's an absolute path and src_root is provided, try that first. Otherwise try treating it as a relative path. The trouble is that if `dep_rel` is something that exists, like `/tmp` to come up with a silly example, this doesn't seem to quite do the right thing.